### DISCUSSION TITLE
HSTS - GitHub Pages

### DISCUSSION THREAD

**Discussion by ksilva-kwn** [community/community#109403] [Other Feature Feedback, Questions, & Ideas]
Hello World!

How can i disable the HSTS from my GitHub Pages?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @ksilva-kwn , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by ksilva-kwn** [reply]
Ohh, thanks for the help bro!!  😉👍

**Comment by DeandrePetersen** [comment]
If you're using a custom domain with GitHub Pages, check your DNS settings. Look for any HSTS-related records, such as Strict-Transport-Security, and remove them if present. This might be set up directly in your DNS provider's control panel.
On the other hand, if you've removed HSTS settings but previously had them configured, browsers that have already visited your site might still have the policy cached. HSTS policies have a max-age directive, which specifies how long the browser should enforce the policy. Typically, this is set to a significant duration, such as a year. You'll need to wait until this duration expires for cached policies to be removed from client browsers.
